apps/portal/examples/multi-option-react-template/content/source/Dockerfile

```
@@ -0,0 +1,14 @@
+FROM node:22-alpine AS build
+WORKDIR /app
+COPY package*.json ./
+RUN npm install
+COPY . .
+RUN npm run build
+
+# Production stage
+FROM nginx:alpine
+COPY --from=build /app/dist /usr/share/nginx/html
+RUN sed -i 's/listen\(  *\)80;/listen ${{ values.port }};/' /etc/nginx/conf.d/default.conf
+EXPOSE ${{ values.port }}
```

Hadolint (2.14.0)

[error] unexpected 'v'
expecting '\', A port optionally followed by the protocol, A range of ports optionally followed by the protocol, a new line followed by the next instruction, or at least one space

(DL1000)